pvr.iptvsimple/addon.xml.in

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.iptvsimple"
-  version="3.5.8"
+  version="3.5.9"
```

Comment:
It’s major.minor.patch so should be 3.6.0

Reply:
updated